luminoso_api/v5_download.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import csv
+from signal import signal, SIGPIPE, SIG_DFL
+from urllib.parse import urlparse
+from tqdm import tqdm as progress
+
+from .v5_client import LuminosoClient
+from .v5_constants import URL_BASE
+
+# Python raises IOError when reading process (such as `head`) closes a pipe.
+# Setting SIG_DFL as the SIGPIPE handler prevents this program from crashing.
+signal(SIGPIPE, SIG_DFL)
```

Comment:
I might be missing something, but is there a reason this can't be handled by regular exception handling?

Reply:
This invocation, and the accompanying comment, came from Sam in `v5_api.py`. I don't know what the issue is either, but I figured that the downloader should do the same in whatever situation this is for.

Reply:
Sam told me over slack that this is not necessary for this script, because it is writing a file rather than writing to stdout, where it might be piped to some other utility.